saleor/data_feeds/google_merchant.py

```diff
@@ -1,266 +0,0 @@
-import csv
-import gzip
-from typing import Iterable
-
-from django.conf import settings
-from django.contrib.sites.models import Site
-from django.contrib.syndication.views import add_domain
-from django.core.files.storage import default_storage
-from django.utils import timezone
-from django.utils.encoding import smart_text
-from django_countries.fields import Country
-
-from ..core.taxes import charge_taxes_on_shipping
-from ..discount import DiscountInfo
-from ..discount.utils import fetch_discounts
-from ..plugins.manager import get_plugins_manager
-from ..product.models import Attribute, AttributeValue, Category, ProductVariant
-from ..warehouse.availability import is_variant_in_stock
```

Comment:
`warehouse.availability.is_variant_in_stock` can be also removed if I'm correct. Also `warehouse.availability.get_available_quantity_for_customer` might not be needed anymore after that.